conf.yaml

```
@@ -1716,8 +1716,8 @@ contents:
           - title:      Ingest Management Guide
             prefix:     en/ingest-management
             current:    7.8
-            branches:   [ master, 7.x, 7.8 ]
-            live:       [ master, 7.x, 7.8 ]
+            branches:   [ master, 7.x, 7.8, 7.9 ]
+            live:       [ master, 7.x, 7.8, 7.9 ]
```

@dedemorton, I think the order here changes the order in the dropdown.

```suggestion
            branches:   [ master, 7.x, 7.9, 7.8 ]
            live:       [ master, 7.x, 7.9, 7.8 ]
```

---

good catch!